index.html

```diff
@@ -285,6 +285,49 @@ <h5>Skills</h5>
           </li>
         </ul>
       </section>
+      <footer class="contact-me">
+        <div class="content">
+          <h3 class="h1 text-bold text-white">Contact me</h3>
+          <p class="text-white">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+          <form
+            class="form"
+            action="https://formspree.io/f/mbjezrpn"
+            method="post"
+          >
+            <input
+              class="input text-medium"
+              type="text"
+              name="name"
+              maxlength="30"
+              placeholder="Name"
+              required
+            />
+            <input
+              class="input text-medium"
+              type="email"
+              name="email"
+              placeholder="Email"
+              required
+            />
+            <textarea
+              class="textarea input"
+              name="message"
+              placeholder="Write your message here"
+              maxlength="500"
+              required
+            ></textarea>
+            <input
+              type="submit"
+              value="Get in touch"
+              class="text-medium button-filled-white"
+            />

```

Comment:
- Hey, please, let's kindly improve the design of our button by adding the interactions to the button to indicate the `hover, active, enabled, disabled` state to improve the user experience when they interact with the button.


![Screenshot from 2023-03-28 15-19-05](https://user-images.githubusercontent.com/59397159/228268323-d5896f97-1e70-4c0c-9c02-b21901d1d93d.png)